src/base/bittorrent/session.cpp

```diff
@@ -4374,23 +4495,29 @@ void Session::handleSessionStatsAlert(libt::session_stats_alert *p)
 {
     qreal interval = m_statsUpdateTimer.restart() / 1000.;
 
-    m_status.hasIncomingConnections = static_cast<bool>(p->values[m_metricIndices.net.hasIncomingConnections]);
+#if LIBTORRENT_VERSION_NUM < 10200
+    const auto& counters = p->values;
+#else
+    const auto& counters = p->counters();
```

Comment:
counters returns a `span<std::int64_t const>`, so you don't really need to take it by reference